config.py

```diff
@@ -0,0 +1,41 @@
+
+import logging
+import uuid
+
+logger = logging.getLogger(__name__)
+
+# try to load config from a top-level module.
+try:
+    import pykitconfig
+except ImportError:
+    pykitconfig = object()
+    logger.info('pykitconfig not found by "import pykitconfig".'
+                ' Using default config.'
+                ' You can create file "pykitconf.py" to define default config for pykit.')
```

Comment:
We need to replace this with k3confloader...
https://github.com/pykit3/k3confloader

Maybe a new test should be added to ensure it reads from the expected config file.

Reply:
> We need to replace this with k3confloader...
> https://github.com/pykit3/k3confloader
> 
> Maybe a new test should be added to ensure it reads from the expected config file.

okay, I see!

Reply:
> We need to replace this with k3confloader...
> https://github.com/pykit3/k3confloader
> 
> Maybe a new test should be added to ensure it reads from the expected config file.

done
I seem to see that zkconf already has ut for conf, do I need to add something else?

Reply:
> zkconf already has ut for conf
What do you mean the "has ut for conf"?

Reply:
> > zkconf already has ut for conf
> > What do you mean the "has ut for conf"?

I mean zkconf already has ut to test reads from the expected Config file, such as test_zkconf.test_default, do I need to add some more?

Reply:
Hm... sorry I missed that. Then there won't be a problem.